frontends/bmd/src/app_root.tsx

```diff
@@ -434,7 +434,8 @@ export function AppRoot({
     return () => {
       clearTimeout(PostVotingInstructionsTimeout.current);
     };
-  }, [showPostVotingInstructions, hidePostVotingInstructions]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [showPostVotingInstructions]);
```

Comment:
Ooh very interesting - how exactly does ignoring updates to `hidePostVotingInstructions` make the timeout actually work?

Reply:
`hidePostVotingInstructions` was updating repeatedly, I believe because it has a dependency on `auth`. So the effect was being cleaned up and the timeout was being cleared repeatedly, thus never hitting the 30s threshold.

Reply:
Got it! Maybe worth adding either a brief explanatory comment or a test to make sure someone doesn't accidentally add the dependency back in